Gemfile

```diff
@@ -53,7 +54,7 @@ gem "rack-cors", "0.4.0", require: "rack/cors"
 
 # CSS
 
-gem "bootstrap-sass", "2.3.2.2"
+gem "bootstrap-sass", "~> 3.3.4"
```

Comment:
Those can still introduce bugs for us by mistake, we can relax this once #6003 is released and the majority of setups uses the `--deployment` flag, but for now I had to troubleshoot errors because users updated with `bundle update` more than once, and I don't want to do that again.